pizza-time/src/components/card/card.js

```diff
@@ -83,7 +85,7 @@ class Card extends React.Component {
       status: "Declined"
     };
     axios
-      .put(`http://localhost:5500/api/events/status/${id}`, newItem)
+      .put("http://"+ this.state.host + "/api/events/status/" + id, newItem)
```

Comment:
Here you could use the template literal syntax `https://${this.state.host}/api/events/status/${id}`